kinto/__main__.py

```diff
@@ -40,12 +46,25 @@ def main(args=None):
     config_file = args['ini_file']
 
     if args['which'] == 'init':
-        if not os.path.exists(config_file):
-            init(config_file)
-        else:
+        if os.path.exists(config_file):
             print("%s already exist." % config_file, file=sys.stderr)
             sys.exit(1)
 
+        backend = args['backend']
+        if not backend:
```

Comment:
you mean prompting in the `init()` method ?

I found it more consistent to have interaction with user along the CLI code rather than along templating.

Do you have any strong argument?